backend/src/main/java/com/bakdata/conquery/models/datasets/SecondaryIdDescription.java

```diff
@@ -20,6 +23,13 @@ public class SecondaryIdDescription extends Labeled<SecondaryIdDescriptionId> im
 
 	private String description;
 
+	/**
+	 * Determines whether this column should be used to group events in {@link com.bakdata.conquery.resources.api.QueryResource.EntityPreview} by the frontend.
+	 *
+	 * This is mostly useful for edge-cases, where certain SecondaryIds are only relevant for {@link com.bakdata.conquery.models.query.queryplan.SecondaryIdQueryPlan}.
+	 */
+	private boolean grouped = true;
+
```

Comment:
Ich bin hier wie auch bei dem Hidden der Meinung, dass die EntityPreview nicht ihre Attribute in viele Klassen reinsprayen sollte, sondern eine Art Lookup haben sollte für was sie wie behandeln soll.

Aber lass uns das gerne besprechen.

Reply:
Hatte ich heute morgen doch schon erwähnt, dass ich so umsetze wie du vorgeschlagen hast :) 

Bereite es gerade vor